### PR TITLE
KFSPTS-12232 supporting longer CFDA numbers

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

**Comment by jhulslander** [comment]
I ended up needing to add a DD override on proposal.  Please don't merge this it yet though.  I need to test account prior year locally yet.

**Comment by jhulslander** [comment]
I was able to validate everything I wanted to validate locally, this can be merged after you feel it is ready to go.
This is ready for review again.

**Comment by jhulslander** [comment]
per stand up reassigning to Nancy

File: src/main/resources/edu/cornell/kfs/module/cg/businessobject/datadictionary/CFDA.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+
+	<bean id="CFDA-cfdaNumber" parent="CFDA-cfdaNumber-parentBean">
+		<property name="maxLength" value="7" />
+		<property name="control">
+			<bean parent="TextControlDefinition" p:size="9" />
+		</property>
+	</bean>
+</beans>
```
[the remark this answers is not in the document]
Comment (jhulslander) [reply src/main/resources/edu/cornell/kfs/module/cg/businessobject/datadictionary/CFDA.xml]: They are different, and I think I need to add an override for cfdaDTO-cfdaNumber , good find

**Comment by jhulslander** [comment]
I will ask the functionals about EzraServiceImpl's default value.

None of the existing unit tests are breaking due to this change, so I think we are OK there.

We typically don't do unit tests on field size checks.  Should we start?

**Comment by nkimble** [comment]
Regarding my question concerning whether the default value needs to change from "XX.XXX" to "XXX.XXX" in class edu.cornell.kfs.module.ezra.service.impl.EzraServiceImpl line #69:

If that default should change, please keep in mind there is existing data in table CG_CFDA_REF_T (one row) and table CG_PRPSL_T (34 rows) that may also need to be updated to keep everything in sync with the constant being changed.

**Comment by jhulslander** [comment]
Per functional feedback, we no longer need to implement this change

**Pull request closed without merging**